roles/contiv_network/tasks/aci_tasks.yml

```diff
@@ -23,6 +23,7 @@
   service: name=aci-gw state=started
   run_once: true
 
+# NOTE: Use Contivctl command when it is available
 - name: set aci mode
-  shell: contivctl global set -nwinfra aci
```

Comment:
ok, I have changed the contivctl release job to do an explicit build after `go get` to allow generating the version code.

@DivyaVavili , please try the recent contivctl release and see if it helps. Hopefully we can continue to use contivctl set of commands. https://github.com/contiv/contivctl/releases/download/v0.0.0-01-31-2016.17-56-53.UTC/contivctl-v0.0.0-01-31-2016.17-56-53.UTC.tar.bz2